src/test/java/by/home/controller/DataReaderTest.java

```diff
@@ -0,0 +1,55 @@
+package by.home.controller;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class DataReaderTest {
+    private final static String TEST_FILE_PATH = "./src/test/resources/testFile.txt";
+
+    @Test
+    public void testReadShouldReturnLinesListWhenFileExists() {
+        // given
+
+        DataReader reader = new DataReader();
+
+        // when
+        List<String> lines = reader.read(TEST_FILE_PATH);
+
+        // then
+        List<String> expectedLines = new ArrayList<String>();
```

Comment:
Arrays.asList
make it class level constant

Reply:
private final static String[] expectedData = new String[] {"2,2,5" , "5,10,-10" , "0,0,2" , "-8,8,5"};

List<String> expectedLines = Arrays.asList(expectedData);

Right?